server/README.md

```diff
@@ -187,6 +187,14 @@ cargo test
 cargo test --features postgres
 ```
 
+- To run sync integration test
+
+Uncomment test attribute, and read description in method comment
+
+https://github.com/openmsupply/open-msupply/blob/e6dc29f1ace2fd22abc5e9a17e368a9d871a6f45/server/server/src/sync/integration_tests/remote_sync_integration_test.rs#L130-L131
+
+See test_remote_syncing in remote_sy
```

Comment:
`remote_sy`...? should there be more to this line?

Reply:
thanks 2b4e713ebf05429515e21c58bc04c52a966746ee